### PR TITLE
Add submission of request to omit and landing page on success

### PR DESCRIPTION
**Pull request by BrandonSharratt**
*Issue:*https://github.com/bcgov/entity/issues/22112

*Description of changes:*
Add all api endpoints to the ui store (get all, get one, push, put)
Set form to disabled while submitting
If submitting is successful route to landing page
if it isn't put error in console and do unlock form (this matches other forms)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-212-lrsrpd7m.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-212-lrsrpd7m.web.app

File: btr-web/btr-main-app/pages/requestToOmit.vue [hunk 1]
```diff
@@ -13,58 +28,88 @@ const { completingPartyRef, omitObscureRef, siBizRef } = storeToRefs(omitIndivid
...
-          :title-expanded="$t('helpTitles.requestOmit.expanded')"
-        >
-          <template #default>
-            <BcrosI18HelperLink translation-path="helpTexts.requestOmit.detail" />
-          </template>
-        </BcrosHelpTip>
+        <span v-if="showSubmission">
+          <h1 class="text-2xl font-bold mb-4" data-cy="request-to-omit-title">
+            {{ $t('labels.requestOmit.header') }}
+          </h1>
```
Comment (patrickpeinanw) [inline btr-web/btr-main-app/pages/requestToOmit.vue:34]: It is better to use `<template v-if="">` so we don't add block-level elements inside inline element

**Review by patrickpeinanw — APPROVED** [review]
Looks good to me! A cypress test is failing but I assume it is working locally :-)

**Pull request merged**